working-groups/design-and-ux.md

```diff
@@ -2,28 +2,78 @@
 layout: text-page
 title: Design & UX
 slug: design-and-ux
+
 ---
 
-**Status**: Active<br>
-**Bottom liner**: Georgia Bullen<br>
-**Working Group members**: Memo Esparza, Perrie Ojemeh, Eriol Fox, Bernard Tyers, Justin W. Flory, Benjamin Nickolls, Richard Littauer <br>
-**How to get involved**: See below.<br>
+# About the squad
```

Comment:
Why the word squad? Seems militaristic, no?

Reply:
@RichardLitt I succumbed to my Markdown linter asking me to put a H1 element in the document, and this is what I decided on. I could remove it or change it to something else, open to suggestions. 🙂

Reply:
I would remove it. I just checked, and there already is an h1 - it renders as the title. See https://sustainoss.org/working-groups/design-and-ux/.

Reply:
Also, regarding my comment about _squad_ - I'm increasingly trying to notice when I use martial metaphors. I don't think it helps build the society I want to live in. See [here](https://www.etymonline.com/word/squad#etymonline_v_21923) for the origins of the word squad. That having been said, huge fan of [the squad](https://en.wikipedia.org/wiki/The_Squad_(United_States_Congress)).

Reply:
I highly recommend this particular podcast for more on this: https://onbeing.org/programs/ocean-vuong-a-life-worthy-of-our-breath/.

Reply:
@RichardLitt OK, I respect this. Today I learned. 🙂 Will drop the line.

Reply:
Resolved in commit 3e9e24f9be4cc62dbdf46b1adcd447386a09cf62